android/src/google/play/release-notes/en-US/default.txt

```diff
@@ -1,10 +1,6 @@
-• New OSM maps data as of November 22
-• Fixed routing between map regions, please test and report to us any issues!
-• Type "water" or "drinking water" in search to find it around yourself
-• Long tap on a POI/bookmark name/address to copy them into clipboard
-• Reworked opening hours UI
-• Restored "maps update is available" indicator
-• Added LINE contacts to the place page
-• Reduced APK size
-• Updated Vulkan driver libraries
-• Updated be, de, es, fi, ru, ua and zh translations
+• New OSM maps data as of 2021-01-03
+• Add rd/st/ct=road/street/court synonyms to the search
+• Do not stop music when announcing directions
+• Improve roads classification
+• Fix non-working FAQ on Android 5 and 6
+• Update localizations
```

Comment:
Added bg, he, sw; updated it, ro, uk, tr, de

Copy most of POI info (name, phone, website...) via single long tap